cfy_manager/components/validations.py

```diff
@@ -360,11 +386,25 @@ def _check_internal_ca_cert():
 
 def _validate_cert_inputs():
     _check_internal_ca_cert()
-    _check_key_and_cert('internal')
-    _check_key_and_cert('postgresql_server')
-    _check_key_and_cert('postgresql_client')
-    _check_key_and_cert('external')
-    _check_key_and_cert('external_ca')
+    for component in (
+        'internal',
```

Comment:
On that same note, we don't have a `internal_ca_path` or `internal_key_password` since the ca internal uses is `ca_cert_path` IIRC.
Same for `external` below.

Reply:
That's OK- it'll ignore their absence and behave as it did before for those cases.